include/boost/gil/image.hpp

```diff
@@ -238,7 +238,10 @@ class image
         swap(_align_in_bytes,  img._align_in_bytes);
         swap(_memory,          img._memory);
         swap(_view,            img._view);
-        swap(_alloc,           img._alloc);
```

Comment:
@mloskot If that can help with the review, this is he important bit (that I would like to be merged in 1.80). The rest can be moved to another PR if necessary.